readthedocs/projects/tasks.py

```diff
@@ -1005,3 +1007,26 @@ def sync_callback(_, version_pk, commit, *args, **kwargs):
     """
     fileify(version_pk, commit=commit)
     update_search(version_pk, commit=commit)
+
+
+@app.task()
+def finish_inactive_builds():
+    delta = datetime.timedelta(hours=1, minutes=30)
+    query = (~Q(state=BUILD_STATE_FINISHED) &
+             Q(date__lte=datetime.datetime.now() - delta))
+
+    builds = Build.objects.filter(query)[:50]
+    for build in builds:
+        build.success = False
+        build.state = BUILD_STATE_FINISHED
+        build.error = (
+            'This build was terminated due to inactivity. If you '
+            'continue to encounter this error, file a support '
+            'request with and reference this build id ({0}).'.format(build.pk)
+        )
```

Comment:
This should be translated, though I'm not sure if we're equipped to actually translate this yet.

Reply:
Equipped? In what sense?

I don't understand how/where the translation are working yet.